storage/storage.c

```diff
@@ -390,34 +388,21 @@ static secbool set_wipe_code(const uint8_t *wipe_code, size_t wipe_code_len) {
   // The format of the WIPE_CODE_DATA_KEY entry is:
   // wipe code (variable), random salt (16 bytes), authentication tag (16 bytes)
```

Comment:
The comment should read:  `  // wipe code (variable), random salt (8 bytes), authentication tag (8 bytes)`
The typo wasn't introduced in this pull request, but we can fix it anyway.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/1afca92292fed9de3476bd0ba90ee1c0fc1600b2

Reply:
- [ ] @